test/general/macros.pass.cpp

```diff
@@ -0,0 +1,54 @@
+// -*- C++ -*-
+//===-- macros.pass.cpp --------------------------------------------------===//

```

Comment:
Please rename to something meaningful in accordance with that test is checking

Reply:
Name variations:
check_of_disable_unused_backend
unused_backend_checking

Reply:
My suggestions is ```backend_inclusion.pass.cpp```.

Reply:
Cool suggestion, it suits best

Reply:
I would say it's `wrong_backend_absence.pass.cpp`. What I don't like with my variant it contains double negation. Probably Dmitriy's suggestion is good enough.